src/main/java/org/alfresco/rest/api/impl/NodesImpl.java

```diff
@@ -245,6 +247,8 @@ public void setNonAttachContentTypes(Set<String> nonAttachWhiteList)
         this.nonAttachContentTypes = nonAttachWhiteList;
     }
 
+    private int directAccessUrlExpiryTime = -1;
```

Comment:
Personally I think we should have a default value.

Reply:
I added the requested 24 hours as the default